utils.py

```diff
@@ -49,14 +78,101 @@ def get_setting(session: Session, key: str, default: Optional[Union[str, bool, L
     return value
 
 
+def message_dialog(msg: str, *, console: bool = False) -> None:
```

Comment:
What if we delegate formatting of the msg to this helper:
```python
message_dialog(msg: str, *, console: bool = False, **kwargs) -> None:
    msg = msg.format(**kwargs)
    # ...
```
this way, we could use it like `message_dialog('Co{what}', what="pilot")`

Reply:
![image](https://user-images.githubusercontent.com/6594915/180190485-b821bb3b-5e25-406f-832a-fccc2b6334ad.png)
Copilot agrees.

Reply:
Done in [eb68648](https://github.com/TheSecEng/LSP-copilot/pull/18/commits/eb686481a56a6a18adcd73d375c28e6e6c35d504)